tests/expected/abort/expected

```diff
@@ -1,4 +1,16 @@
-FAILURE\
-a panicking function std::process::abort is invoked
-SUCCESS\
-a panicking function std::process::abort is invoked
+Status: UNREACHABLE\
+Description: ""This is unreachable""\
+in function main
+
+Status: FAILURE\
+Description: "Function abort() was invoked"\
+in function std::process::abort
+
+Status: SUCCESS\
```

Comment:
I was expecting this to be UNREACHABLE, but it isn't because `std::exit` calls `panic` which we currently don't override. If we want it to be reported as UNREACHABLE (I think it should, but I'm not sure), then we can change the `panic(msg)` to `assert!(false, msg)`.

Reply:
We can't. It won't compile because of the never return. I think we should instrument panic inside the compiler to have a reach check.

Reply:
I see.